Lib/fontTools/cffLib/specializer.py

```diff
@@ -97,10 +100,12 @@ def vmoveto(args):
 
 	@staticmethod
 	def rlineto(args):
+		if len(args) == 0: raise ValueError(args)
```

Comment:
I do too, but decided to err on the conservative side because checks like these sometimes contain nuances.